packages/react-reconciler/src/ReactFiberReconciler.old.js

```diff
@@ -107,10 +108,7 @@ type DevToolsConfig = {|
   // Note: this actually *does* depend on Fiber internal fields.
   // Used by "inspect clicked DOM element" in React DevTools.
   findFiberByHostInstance?: (instance: Instance | TextInstance) => Fiber | null,
-  // Used by RN in-app inspector.
-  // This API is unfortunately RN-specific.
-  // TODO: Change it to accept Fiber instead and type it properly.
-  getInspectorDataForViewTag?: (tag: number) => Object,
+  rendererConfig?: RendererInspectionConfig,
```

Comment:
You also need to change `ReactFiberReconciler.new.js`.

Reply:
Is that still in a branch? I dont see it on master?

Reply:
Ignore me. You're right.